roles/swift-common/tasks/main.yml

```diff
@@ -76,6 +76,13 @@
     - logging
   when: logging.enabled|default('True')|bool
 
+- name: change swift log file permissions
+  file: path={{ item }} mode=0644 owner=swift group=swift state=touch
+  with_items:
+    - /var/log/swift/account.log
+    - /var/log/swift/container.log
+    - /var/log/swift/object.log  
+
```

Comment:
currently these logs should be created with 0640 owned by syslog:adm ( inherited from the directory perms https://github.com/blueboxgroup/ursula/blob/master/roles/swift-common/tasks/logging.yml#L2-L3 )

does settings it to 0644 by swift:swift improve security ?   How will syslog write to them without write permissions ?